internal/core/textlayout/sharedparley.rs

```diff
@@ -1213,6 +1221,66 @@ pub fn draw_text(
     }
 }
 
+#[cfg(feature = "experimental-rich-text")]
+pub fn handle_links(
+    scale_factor: ScaleFactor,
+    text: Pin<&dyn crate::item_rendering::RenderText>,
+    font_request: Option<FontRequest>,
+    size: LogicalSize,
+    cursor: PhysicalPoint,
+) -> Option<std::string::String> {
+    let str = text.text();
+
+    let layout_text = Text::RichText(parse_markdown(&str));
+
+    let (horizontal_align, vertical_align) = text.alignment();
+
+    let layout = layout(
+        layout_text,
+        scale_factor,
+        LayoutOptions {
+            horizontal_align,
+            vertical_align,
+            max_height: Some(size.height_length()),
+            max_width: Some(size.width_length()),
+            stroke: None,
+            font_request,
+            text_wrap: text.wrap(),
+            text_overflow: text.overflow(),
+            selection: None,
+            selection_foreground_color: None,
+            link_color: crate::Color::from_rgb_u8(64, 64, 255),
```

Comment:
Idea: perhaps the bultin element could have a link color property that the compiler default binds to a property from the Palette. Doesn't have to be in this PR, but that way we could accommodate things like dark mode vs light mode.